webapp/src/actions/index.js

```diff
@@ -80,4 +81,28 @@ export function performSearch(terms) {
 
         return dispatch(searchPostsWithParams(teamId, {terms, is_or_search: false, include_deleted_channels: viewArchivedChannels, page: 0, per_page: 20}, true));
     };
-}
\ No newline at end of file
+}
+
+export function requeueItem(postId) {
+    return async (dispatch, getState) => {
+        const command = `/agenda requeue ${postId}`;
```

Comment:
Since we plan to support other commands in the future like this, I propose we be specific here that we are passing a post id https://github.com/mattermost/mattermost-plugin-agenda/pull/89#discussion_r741641717

```suggestion
        const command = `/agenda requeue post ${postId}`;
```

Reply:
@mickmister  Updated code as per feedback